semantic-source/semantic-source.cabal

```diff
@@ -0,0 +1,88 @@
+cabal-version:       2.4
+
+name:                semantic-source
+version:             0.0.0.0
+synopsis:            Types and functionality for working with source code
+description:         Types and functionality for working with source code (program text).
+homepage:            https://github.com/github/semantic/tree/master/semantic-source#readme
+bug-reports:         https://github.com/github/semantic/issues
+license:             MIT
+license-file:        LICENSE
+author:              The Semantic authors
+maintainer:          opensource+semantic@github.com
+copyright:           (c) 2019 GitHub, Inc.
+category:            Data
+build-type:          Simple
+stability:           alpha
+extra-source-files:
+  README.md
+
+
+tested-with:
+  GHC == 8.6.5
+
+common common
+  default-language: Haskell2010
+  ghc-options:
+    -Weverything
+    -Wno-missing-local-signatures
+    -Wno-missing-import-lists
+    -Wno-implicit-prelude
+    -Wno-safe
+    -Wno-unsafe
+    -Wno-name-shadowing
+    -Wno-monomorphism-restriction
+    -Wno-missed-specialisations
+    -Wno-all-missed-specialisations
+    -Wno-star-is-type
+
+library
+  import: common
+  exposed-modules:
+    Source.Loc
+    Source.Range
+    Source.Source
+    Source.Span
```

Comment:
IMO we should format `.cabal` files this way in general. Aligning crap to the right of labels is terrible.